<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutChequeInterfaceClass</class>
 <widget class="QDialog" name="AjoutChequeInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajout d'un compte Chèque</string>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>171</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditNoCompte">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditTauxInteret"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditSolde"/>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBoxNombreTransactions"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditTauxInteretMinimum"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditDescription"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>30</y>
     <width>144</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelNoCompte">
      <property name="text">
       <string>Numéro de Compte :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelTauxInteret">
      <property name="text">
       <string>Taux d'intérêt :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelSolde">
      <property name="text">
       <string>Solde :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelNombreTransactions">
      <property name="text">
       <string>Nombre de transactions :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelTauxInteretMinimum">
      <property name="text">
       <string>Taux d'intérêt minimum :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelDescription">
      <property name="text">
       <string>Description :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
